net-misc/nextcloud-client/nextcloud-client-3.14.2.ebuild

```
@@ -0,0 +1,118 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Desktop Syncing Client for Nextcloud"
+HOMEPAGE="https://github.com/nextcloud/desktop"
+SRC_URI="
+	https://github.com/nextcloud/desktop/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz
+"
+S="${WORKDIR}/desktop-${PV/_/-}"
+
+LICENSE="CC-BY-3.0 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="doc dolphin nautilus test webengine"
+RESTRICT="!test? ( test )"
+
+# slot op for qtqui as this package uses private API parts of qtqui
+# src/gui/generalsettings.cpp:#include <private/qzipwriter_p.h>
+RDEPEND="
+	>=dev-db/sqlite-3.34:3
+	>=dev-libs/openssl-1.1.0:0=
+	dev-libs/glib:2
+	dev-libs/qtkeychain:=[qt6(+)]
+	dev-qt/qt5compat:6
+	dev-qt/qtbase:6[dbus,gui,network,sql,widgets]
+	dev-qt/qtdeclarative:6[widgets]
+	dev-qt/qtsvg:6
+	dev-qt/qtwebsockets:6
+	net-libs/libcloudproviders
+	kde-frameworks/karchive:6
+	kde-frameworks/kguiaddons:6
+	sys-libs/zlib
+	dolphin? (
+		kde-frameworks/kcoreaddons:6
+		kde-frameworks/kio:6
+	)
+	nautilus? ( dev-python/nautilus-python )
+	webengine? ( dev-qt/qtwebengine:6[widgets] )
+"
+DEPEND="
+	${RDEPEND}
+	dev-qt/qtbase:6[concurrent,xml]
+	|| ( gnome-base/librsvg media-gfx/inkscape )
+	doc? (
+		dev-python/sphinx
+		dev-tex/latexmk
+		dev-texlive/texlive-latexextra
+		virtual/latex-base
+	)
+	test? (
+		dev-util/cmocka
+	)
+"
+BDEPEND="
+	dev-qt/qttools:6[linguist]
+	dolphin? ( kde-frameworks/extra-cmake-modules )
```

Similarly, you need a minimum ECM version that will guarantee Qt6/KF6 aware modules. Build system will likely define that, however, with current `ecm.eclass` baseline (5.106.0) you'll definitely be safe.
```suggestion
	dolphin? ( >=kde-frameworks/extra-cmake-modules-5.106.0 )
```